packages/widget.api.unstable/src/messages/widgetMessages.ts

```diff
@@ -62,11 +75,23 @@ export namespace WidgetMessage {
   export interface EmitEvent<C extends WidgetConfig<C["parameters"]>>
     extends WidgetBaseMessage<"widget.emit-event", Payload.EmitEvent<C>>
   {}
+
+  /**
+   * Temporary fetch proxy
+   * Will be removed in favor of server side proxy
+   */
+  export interface _unstable_FetchRequest extends
+    WidgetBaseMessage<
+      "widget._unstable_fetch-request",
```

Comment:
To match the host events
```suggestion
      "widget._unstable.fetch-request",
```